### ISSUE TITLE
Combobox : aria expanded true when empty suggestions

### ISSUE THREAD

**Issue by MrX8503** [reach/reach-ui#347]
### Current Behavior
On input change and no suggestions, combobox aria-expanded is set to true

### Expected behavior
combobox aria-expanded should be false when there are no suggestions

### Reproducible example

[Example](https://ui.reach.tech/combobox/)
Go to Server Side example and type in "1".

**Comment by rubenmoya** [comment]
It also happens with the Client Side example

**Comment by chancestrickland** [comment]
The popover should still be expanded when no results are present, as there should be some message informing the user that there are no matching results. I'll make sure to update the docs to reflect that!
